source/support/how-to-update-the-support-rota.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: How to update the support rota
+weight: 6
+last_reviewed_on: 2023-11-06
+review_in: 6 months
+---
+
+All designers and developers on the GOV.UK Design System team are on the support rota. The rota is updated monthly.  
+
+# Updating the rota
+
+The Design System team events calendar is used to host the support rota. Most members of the team can add new events to this calendar.
+
+Add an all day event to the calendar stating who is on support. For example:
+
+> Kelly on support
+
+Repeat this for the following month, rotating who is on support each day. Pay attention to non-working days and planned leave when allocating. 
+
+Support is not required on no meetings days.
```

Comment:
Should this be that support is not required on team ceremony days?

Reply:
Thanks, yes it should be. Getting myself confused.... 🤪